Resources/Private/Templates/Module/Search.html

```diff
@@ -7,7 +7,7 @@ <h1>{f:translate(key:'search')}</h1>
 		<f:form action="search" controller="Module" class="form-group">
 			<div class="form-group">
 				<div class="input-group">
-					<f:form.textfield name="word" value="{word}" class="form-control" placeholder="Label"/>
+					<f:form.textfield name="word" value="{word}" class="form-control" placeholder="{f:translate(key:'key')}"/>
```

Comment:
The should be the the word "Label", or something like `{f:translate(key:'label')}`.
```suggestion
					<f:form.textfield name="word" value="{word}" class="form-control" placeholder="Label"/>
```

Reply:
The search is done within the translations but not the key, correct? I agree then, `key` is the wrong text then. Perhaps sth. like "translation" (en), "Übersetzung" (de)

Reply:
I'd prefer "Label" for both languages, or maybe just "Text"..

Reply:
Hmm. I prefer text. If you find time to adjust today, feel free to do so. I may find time somewhen this weekend.